UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -169,5 +169,12 @@ public void NegativeLengthToStonePoundsReturnsCorrectValues()
             Assert.Equal(-2.0, feetInches.Feet);
             Assert.Equal(-1.0, feetInches.Inches);
         }
+
+        [Fact]
+        public void LengthCreatedInSISystem()
```

Comment:
Can we have tests for the imperial units? Those seem to be added in JSON already.